### ISSUE TITLE
Source links should have some accompanying description

### ISSUE THREAD

**Issue by LivingInfinite** [MrSujano/Sources#3]
After watching a video, I may be interested in a certain topic, but when I check the sources on github it's all hyperlinks with no description. Some of them can be deduced by carefully looking at the URL or by the order they appear in, but it would be a lot nicer if there were some accompanying description for them. For example the title of the article or youtube video. 

More ideally these links would also be directly in the description on the youtube video rather than having to come to github to see them.

**Comment by Androxilogin** [comment]
Absolutely. This is what always bothers me to the point I don't even look at his sources, I just go search for it. Unless I can't find it somewhere else. Well, this is the first time I've actually come to the github (odd place for what could essentially be a pastebin document rather than a coding environment) but in the description of videos they have been all jumbled together too. 

I appreciate all that he does, but this is a mess. Given the resources here, I don't understand why they would not be hyperlinks.

Also, the reason I came here was to find the link to **[The Halloween Island](https://www.smwcentral.net/?p=section&a=details&id=21388&r=0)** which he said just released. The latest version was from 2019. In the sources, it was impossible to find out without clicking through them what held the link to it because they were unnamed YouTube videos.

**Comment by Loyla486** [comment]
I am in full agreement as well, when I look at the sources and I want to go to a link that belongs to a story, it is more difficult than it should be to find the link(s) that belong to that story.

**Comment by MrSujano** [comment]
Appreciate the feedback. I can't promise anything, but I will look into incorporating this recommendation in the future. As I'm only a one-man show and do this as a hobby, it may take a bit to figure out as my time is limited and this would introduce additional steps/time in my process. I'll keep this open until I figure out an efficient process.

**Comment by Loyla486** [comment]
> Appreciate the feedback. I can't promise anything, but I will look into incorporating this recommendation in the future. As I'm only a one-man show and do this as a hobby, it may take a bit to figure out as my time is limited and this would introduce additional steps/time in my process. I'll keep this open until I figure out an efficient process.

I notice you spaced them out for "2025.10.5", which is better than last time, however there are still no titles. It shouldn't be too hard to add titles.

**Comment by LeeNX** [comment]
I think @MrSujano is doing awesome things, thank you a ton!!

Spaces are cool, but I think asking for titles is a bit much, unless we would prefer only getting updates every second day or something, because adding a header I think would be as much work as adding a transcript. I believe that the updates are very much ad-hoc and sometimes does not even have a headline.  
  
If I could add my 2c, which I am thinking if I had the time to do this(and I don't think my take would as awesome), could possible be adding something like item 1 to x in the timestamp for the video and possible sync that number in the github release. That would make looking up the URL note easier and would not need too much effort to make up a title for each feedback item.

I was thinking it might be possible to write a script that could pull the title from the noted URL and put that into the release notes, but things like chirps and short message updates, don't alway have a title, so less complete. Not sure having half with titles and other without, somebody will complain.

Dig you work @MrSujano and appreciate all the hard work it takes to make the videos. We are so lucky!

**Comment by Loyla486** [comment]
> If I could add my 2c, which I am thinking if I had the time to do this(and I don't think my take would as awesome), could possible be adding something like item 1 to x in the timestamp for the video and possible sync that number in the github release. That would make looking up the URL note easier and would not need too much effort to make up a title for each feedback item.


I was thinking something similar to this as well, just copy and paste the headlines used for the timestamps and use them as titles. Though it is ultimately not a huge deal either way, I would just like an easier way to navigate the sources, especially when there is a story of interest.

**Comment by MrSujano** [comment]
Added timestamps and separation in the sources. I'll try to be consistent with these, but it does add to my overall extremely time-limited process. I can't promise I will be consistent, but I'll do my best.

**Comment by Loyla486** [comment]
Thanks, appreciate your work @MrSujano!
